lib/fs.js

```diff
@@ -1615,8 +1615,15 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (options === undefined || options === null)
```

Comment:
You can probably drop the `null` check, as it will work fine with `Object.create()`